rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -743,11 +767,7 @@ extern "C" rmw_node_t * rmw_create_node(
   const dds_domainid_t did = DDS_DOMAIN_DEFAULT;
 #endif
 
-  if (security_options == nullptr) {
-    RCUTILS_LOG_ERROR_NAMED(
-      "rmw_cyclonedds_cpp", "rmw_create_node: security options null");
-    return nullptr;
-  }
+  RCUTILS_CHECK_ARGUMENT_FOR_NULL(security_options, nullptr);
```

Comment:
Do you want to include the message here?

Reply:
No, I believe the macro response of "security_options argument is null" is sufficient.